src/main/java/com/epam/test_generator/services/SuitService.java

```diff
@@ -61,4 +69,50 @@ public void removeSuit(long suitId) {
 
         caseVersionDAO.delete(suit.getCases());
     }
+
+    public void updateSuitRowNumber(List<SuitRowNumberUpdate> rowNumberUpdates) {
+        if (rowNumberUpdates.isEmpty()) {
+            throw new IllegalArgumentException("The list has not to be empty");
+        }
+
+        for (SuitRowNumberUpdate s : rowNumberUpdates) {
+            if(s.getId() == null)
+                throw new IllegalArgumentException("Id has not to be null");
+        }
+
+        final List<SuitRowNumberUpdate> sortedRowNumbers = ascSortById(rowNumberUpdates);
+        final List<Long> ids = retrieveIdsFromDTO(sortedRowNumbers);
+        final List<Suit> suits = getASCSortedSuitsById(ids);
+
+        if (ids.size() != suits.size()) {
+            throw new NotFoundException("No such Suit");
+        }
+        updateRowNumbers(sortedRowNumbers, suits);
+
+        suitDAO.save(suits);
+    }
+
+    private void updateRowNumbers(List<SuitRowNumberUpdate> sortedRowNumbers, List<Suit> suits) {
+        for (int i = 0; i < suits.size(); i++) {
+            final Integer newRowNumber = sortedRowNumbers.get(i).getRowNumber();
+            suits.get(i).setRowNumber(newRowNumber);
+        }
+    }
+
+    private List<Suit> getASCSortedSuitsById(List<Long> ids) {
```

Comment:
Ascending order is default sorting order, no point to name method as ascending explicitly.